ocs_ci/ocs/ui/base_ui.py

```diff
@@ -334,6 +357,21 @@ def navigate_overview_page(self):
             self.choose_expanded_mode(mode=True, locator=self.page_nav["Home"])
         self.do_click(locator=self.page_nav["overview_page"])
 
+    def navigate_odf_overview_page(self):
+        """
+        Navigate to OpenShift Data Foundation Overview Page
+
+        """
+        if Version.coerce(self.ocp_version) >= Version.coerce("4.9"):
+            ocs_version = version.get_semantic_ocs_version_from_config()
+            if ocs_version >= version.VERSION_4_9:
```

Comment:
If no action is taken in 'else' , the conditional statements can be removed. Use appropriate version checks from where this function is called.